test/unit/lib/Dequeuer.test.js

```diff
@@ -52,7 +52,7 @@ test('Dequeuer.executeCallback(): should ack processed message', async (t) => {
   const callback = async () => true;
   const callbackSpy = sinon.spy(callback);
 
-  // Prepare random message to make dequeur think it fot it from Rabbit.
+  // Prepare random message to make dequeuer think it fot it from Rabbit.
```

Comment:
I think there's still a typo on this line:


```suggestion
  // Prepare random message to make dequeuer think it got it from Rabbit.
```